libdnf/base/goal.cpp

```diff
@@ -155,23 +153,23 @@ void Goal::add_rpm_upgrade(const libdnf::rpm::Package & rpm_package) {
 
 void Goal::add_rpm_upgrade(const libdnf::rpm::PackageSet & package_set) {
     libdnf::rpm::solv::IdQueue ids;
-    for (auto package_id : *package_set.pImpl) {
+    for (auto package_id : *package_set.p_impl) {
         ids.push_back(package_id.id);
     }
     p_impl->upgrade_rpm_ids.push_back(std::move(ids));
 }
 
 void Goal::add_rpm_upgrade(const libdnf::rpm::SolvQuery & query) {
```

Comment:
Minor - I suggest that the whole function could be removed

Reply:
Removed.